packages/venia-concept/src/ServiceWorker/registerRoutes.js

```diff
@@ -9,17 +9,7 @@ export default function() {
     const catalogCacheHandler = createCatalogCacheHandler();
 
     workbox.routing.registerRoute(
-        '/',
-        new workbox.strategies.StaleWhileRevalidate()
-    );
-
-    workbox.routing.registerRoute(
-        new RegExp('\\.html$'),
-        new workbox.strategies.NetworkFirst()
-    );
-
-    workbox.routing.registerRoute(
-        new RegExp('/.\\.js$'),
+        new RegExp('(robots.txt|favicon.ico|manifest.json)'),
```

Comment:
Quick regex gotcha: that `.` actually matches on any character. If you want the literal `.`, it needs to be escaped: `\.`.

Doesn't prevent a 👍 , just an fyi :)